README.md

```diff
@@ -297,7 +312,7 @@ The following are a list of the experimental explainers available in the communi
 The following two sections demonstrate how you can get global and local feature importance values. Local measures focus on the contribution of features for a specific prediction (e.g., why the model predicted an 80% chance of breast cancer for Mary?), whereas global measures take all predictions into account (Overall, what are the top K important features in predicting a high risk for breast cancer?):
 ## Overall (Global) feature importance values
 
-Get the global feature importance values.
+Get the aggregate feature importance values.
```

Comment:
We're still using global elsewhere in this documentation - should we stick to one term?

Reply:
Done.